swap_meet/vendor.py

```diff
@@ -1,2 +1,72 @@
-class Vendor:
-    pass
\ No newline at end of file
+class Vendor():
+   def __init__(self, inventory = None):
+      
+      self.inventory = [] if inventory is None else inventory
+
+   def add(self, item):
+      self.inventory.append(item)
+      return item
+
+   def remove(self,item):
+      if not item in self.inventory:
+         return False
+      self.inventory.remove(item)
+      return item
+
+   def get_by_id(self, id):
+      for item in self.inventory:
+         if item.id == id:
+               return item
+      return None
+   
+   def get_by_category(self, category):
+      items_with_category = []
+      if not self.inventory:
+         return items_with_category
```

Comment:
We can get away with removing this check because if `self.inventory` was an empty list, then line 26 wouldn't have anything to iterate over and it would just go on to line 29 where it would return `items_with_category` which would still be an empty list. 

```suggestion

```